pom.xml

```diff
@@ -1830,6 +1836,7 @@
         </profile>
 
         <profile>
+            <!-- TODO: Should this profile still exist now that we have a minimum of 17?-->
```

Comment:
I'd opt to remove the profile and include the plugin to the default plugins.

Reply:
Agreed, I've moved the plugin.